lib/python/picongpu/extra/input/createBunch_example.ipynb

```diff
@@ -200,7 +198,7 @@
     "\n",
     "# shift particles to new center position\n",
     "x_PIConGPU = x + center_pos_x\n",
-    "y_PIConGPU = y - np.mean(y) + center_pos_y\n",
+    "y_PIConGPU = y + center_pos_y\n",
```

Comment:
Just a question:
Why this change?

Reply:
It seemed redundant to me as the bunch is already distributed around 0 in the example script so I threw it away for simplicity.

Reply:
Thanks for the calrification